packaging/test_irods_client_rest_cpp.py

```diff
@@ -529,3 +529,61 @@ def test_changing_passwords_is_supported__issue_99(self):
         self.user.assert_icommand(['iinit', old_password])
         self.user.assert_icommand(['ils', '-ld'], 'STDOUT', [self.user.session_collection])
 
+    def test_replicate_data_object():
+        with session.make_session_for_existing_admin() as admin:
+            token = irods_rest.authenticate('rods', 'rods', 'native')
+            logical_path = '/tempZone/home/rods/hello.cpp'
+            new_resc_name = 'newResc'
+            ## create a data object on default resource
+            admin.assert_icommand(["itouch", logical_path])
+            ## create new resource
+            lib.create_ufs_resource(new_resc_name, admin)
```

Comment:
We need to remove the resource after completion. Not doing so will result in failures.

Reply:
This can be done with `lib.remove_resource`. May want to use a try-finally block here to ensure the cleanup as well.

Reply:
`lib.remove_resource` I believe uses `assert_icommand`... which we wouldn't usually want in a `finally` block.

Reply:
> `lib.remove_resource` I believe uses `assert_icommand`... which we wouldn't usually want in a `finally` block.

This is true.